src/config.h

```diff
@@ -48,6 +48,12 @@ typedef enum {
     BSM_PS1 = 5,
 } BAR_SHOW_MODE;
 
+typedef enum {
+    TLM_FULL = 0,
+    TLM_SEMI = 1,
+    TLM_NO = 2,
```

Comment:
I can't explain it but to me `TLM_NONE` sounds more fitting.

Reply:
Ok I renamed it internally to `TLM_NONE`. I left it as `No` in the config tool because it's referred to as `No lock`.